examples/device/ep/nixl_ep/buffer.py

```
@@ -728,17 +728,18 @@ def ht_combine(
 
     def update_mask_buffer(self, rank_to_mask: int, mask: bool = False):
```

Ruff (0.15.14)

[warning] Boolean-typed positional argument in function definition

(FBT001)